src/docs/index.js

```diff
@@ -67,49 +68,33 @@ const FormContaner = styled.form`
   padding: 10px;
 `;
 
+const Logger = () => <Logs data={data}
+                         path="path>path?path"
+                         onClear={() => console.log('Clear all')}
```

Comment:
suggest to create a separate functions, cause you pass new functions as a props each time

Reply:
I just drop this function because this needed only for QA

Reply:
I see